newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -466,6 +466,7 @@ export const AssetDetails = React.forwardRef<Props, AssetDetailsInterface>(
                               makeFirstLetterUppercase(animation.name) ||
                               t`Default` // Display default for animations with no name.
                             }
+                            primaryTextIsUserDefined={!!animation.name}
```

Comment:
I don't understand why it depends on the presence of an animation name

Reply:
I'll update the code so that it's clearer.
it's because if animation.name is not defined, we replace the text by a translatable one (otherwise we can't/shouldn't translate it)